docs/02-guides/server-side-events/integrations/google-analytics.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Google Analytics
+description:
+  Implement server-side events with Google Analytics Measurement Protocol.
+---
+
+<p>{frontMatter.description}</p>
+
+## Introduction
+
+Google Analytics provides a JS SDK to push events to their server. To do
+server-to-server communication, you need to use Google Analytics Measurement
+Protocol.
+
+## Setup Google Analytics
+
+To use Google Analytics Measurement Protocol, you need to get your API Secret
+and your measurement ID. Please follow the
+[Google Analytics Measurement Protocol Documentation](https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference?client_type=gtag#payload_query_parameters)
+to learn how to get them.
```

Comment:
**thought:** I feel like this information doesn't deserve its own section.

**suggestion:** Put this in a `info` block in the section below.

Reply:
Fixed with your suggestion